communications/newsletters/newsletter_14_May2020.md

```diff
@@ -0,0 +1,192 @@
+
+## Upcoming workshops and events
+
+### Boost Research Reproducibility with Binder Workshop
+
+In collaboration with [UK DRI](https://ukdri.ac.uk/), _The Turing Way_ is hosting a half-day workshop on Binder on 11 June 2020, from 13:00 to 17:00.
+Binder allows you to share your software from GitHub in a single clickable link, making it easier for users to not worry about the computational environments, dependencies and platforms.
+In this workshop, we will discuss reproducible computing environments, show examples of others’ projects in myBinder.org and help you learn how to prepare a binder-ready project. 
+At the end of the workshop, you will be able to take some of your own content (in Jupyter notebook or scripts) and prepare it so that it can be used by others on myBinder.org.
+More information is available on the [registration page online](https://www.eventbrite.co.uk/e/workshop-boost-research-reproducibility-with-binder-registration-105228216428).
+If you have previous experience with Binder and would like to join as a helper, please reach out to us by emailing theturingway@gmail.com.
+
+### Google Seasons of Docs
+
+INCF is taking part in this year's [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants) as an open source organisation and _The Turing Way_ is listed as one of its projects.
+During its exploration phase from May 11 to June 8, we invite the professional technical writers to connect with us and discuss the project ideas that can allow them to bring in their perspectives and unique skill set.
+_The Turing Way_ involves a large community of volunteers who edit, review and maintain its resources online on GitHub.
+As a book project, we can benefit tremendously from the support of experienced individuals who can provide editorial support and related guidance in our project.
+We have compiled some recommendations that can be used as the starting points for the GSOD candidates.
+Please find details on [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/1075), and feel free to comment there if any further information will be useful.
+
+### Data Feminism Book and “Challenging Power in Data Science” seminar 
+
+[Data Feminism](https://mitpress.mit.edu/books/data-feminism) is a recent book by Catherine D'Ignazio and Lauren F. Klein that explains the concept of data feminism as a way of thinking about data science and its communication informed by decades of intersectional feminist activism and critical thought. 
+They will be giving a talk hosted by The Alan Turing Institute to discuss how challenges to the male/female binary can challenge other hierarchical classification systems; how an understanding of emotion can expand our ideas about effective data visualization; and how the concept of “invisible labor” can expose the significant human efforts required by our automated systems. 
```

Comment:
You use challenges twice in this sentence, is there another word we could use?

Reply:
```suggestion
They will be giving a talk hosted by The Alan Turing Institute to discuss how challenging the male/female binary system can help investigate other hierarchical classification systems; how an understanding of emotion can expand our ideas about effective data visualization; and how the concept of “invisible labor” can expose the significant human efforts required by our automated systems. 
```

Reply:
I took that from the eventbrite description, rephrased that now.